examples/visual-tests/amp-ad/amp-fie-doubleclick.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
i think the test is doubleclick irrelevant? let's rename the file

/amp-ad/fie.js